tests/integration/test_engine/test_events/test_events.py

```diff
@@ -1,83 +1,97 @@
-import os
-import pytest
-import socket
-import struct
-import shutil
-
-from wazuh_testing.tools.configuration import get_test_cases_data
-from wazuh_testing.modules.engine import event_monitor as evm
-from wazuh_testing.modules import engine
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
 
+           Created by Wazuh, Inc. <info@wazuh.com>.
 
-# Reference paths
-TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
-TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
-# Configuration and cases data
-t1_cases_path = os.path.join(TEST_CASES_PATH, 'cases_engine_logs.yaml')
+type: integration
 
-# Engine events configurations
-t1_configuration_parameters, t1_configuration_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+brief: These tests will check that the events are correctly handled by the engine module.
 
-# vars
-events = t1_configuration_metadata[0]['events']
-expected_outputs = t1_configuration_metadata[0]['engine_outputs']
+components:
+    - engine
 
+suite: test_events
 
-@pytest.fixture(scope='module')
-def create_kvdb():
-    if not os.path.isdir(engine.KVDB_PATH):
-        print('INFO: Creating the KVDB database')
-        command = f"{engine.ENGINE_BUILD_PATH}/main kvdb -p {engine.OUTPUT_FOLDER} -n win-security-categorization " \
-                  f"-i {engine.KVDB_WIN_INPUT} -t json"
-        print(f"INFO: Running {command}")
+targets:
+    - manager
 
-        engine.run_local_command_printing_output(command)
-    else:
-        print(f"INFO: The KVDB already exists in {engine.KVDB_PATH}")
+daemons:
+    - wazuh-engine
 
-    yield
+os_platform:
+    - linux
 
-    print('INFO: Removing KVDB files')
-    shutil.rmtree('/tmp/win-security-categorization/', ignore_errors=False, onerror=None)
+os_version:
+    - Ubuntu Focal
```

Comment:
What about Centos, ALAS, Debian, Suse, RHEL? Is it only for Ubuntu Focal?

Reply:
As the engine has only been tested on this operating system, this list will be updated as we test it on more systems.